include/bredis/impl/connection.ipp

```diff
@@ -99,37 +101,33 @@ void Connection<NextLayer>::async_read(DynamicBuffer &rx_buff,
 }
 
 template <typename NextLayer>
-void Connection<NextLayer>::write(const command_wrapper_t &command) {
-    namespace asio = boost::asio;
-    namespace sys = boost::system;
-
-    auto str = boost::apply_visitor(command_serializer_visitor(), command);
-    BREDIS_LOG_DEBUG("async_write >> " << str);
-    auto const output_buf = asio::buffer(str.c_str(), str.size());
-    asio::write(stream_, output_buf);
-}
-
-template <typename NextLayer>
 void Connection<NextLayer>::write(const command_wrapper_t &command,
                                   boost::system::error_code &ec) {
     namespace asio = boost::asio;
-    namespace sys = boost::system;
-
     auto str = boost::apply_visitor(command_serializer_visitor(), command);
     BREDIS_LOG_DEBUG("async_write >> " << str);
     auto const output_buf = asio::buffer(str.c_str(), str.size());
     asio::write(stream_, output_buf, ec);
 }
 
 template <typename NextLayer>
+inline void Connection<NextLayer>::write(const command_wrapper_t &command) {
```

Comment:
I wouldn't use the `inline` keyword. Let the compiler decide, they usually do a better job than humans.